woocommerce/woocommerce-android
# Feature/8955 close account UI

Part of: #8955 


### Description

Upcoming policies from Google require us to provide a way to close accounts. More on that in the linked issues. This PR adds the UI needed to close an account from settings. 

No request is fired. The loading is simply a delay for testing purposes. 

The UI for this feature is replicating iOS behavior with some slight changes. 

### Testing instructions

- Navigate to settings 
- Click close account
- Verify the behavior is as shown in the screen recording

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/2663464/33f47eee-c179-4b8a-a909-5963e6ea4c4a
